Content.Client/MassMedia/Ui/ArticleEditorPanel.xaml.cs

```diff
@@ -0,0 +1,91 @@
+﻿using Content.Client.Message;
+using Content.Client.Stylesheets;
+using Robust.Client.AutoGenerated;
+using Robust.Client.UserInterface;
+using Robust.Client.UserInterface.Controls;
+using Robust.Client.UserInterface.CustomControls;
+using Robust.Client.UserInterface.XAML;
+using Robust.Shared.Utility;
+
+namespace Content.Client.MassMedia.Ui;
+
+[GenerateTypedNameReferences]
+public sealed partial class ArticleEditorPanel : Control
+{
+    public event Action? PublishButtonPressed;
+
+    private bool _preview = false;
+
+    public ArticleEditorPanel()
+    {
+        RobustXamlLoader.Load(this);
+
+        ButtonPublish.StyleClasses.Add(StyleBase.ButtonOpenLeft);
+        ButtonPublish.StyleClasses.Add(StyleNano.StyleClassButtonColorGreen);
```

Comment:
I think StyleClasses can be in the xaml? ButtonPublish.

Reply:
I don't quite remember anymore but I tried setting them in xaml first but there was a problem with that.